docs/user/index.rst

```diff
@@ -92,126 +92,76 @@ Open Source and User Focused |:heartbeat:|
 
 You can find out more about our all the :doc:`features </reference/features>` in these pages.
 
-First steps
------------
-
-Are you new to software documentation?
-Are you looking to use your existing docs with Read the Docs?
-Learn about documentation authoring tools such as Sphinx and MkDocs
-to help you create fantastic documentation for your project.
-
-* **Tutorial**: :doc:`/tutorial/index`
-
-* **Getting started**:
-  :doc:`With Sphinx </intro/getting-started-with-sphinx>` |
-  :doc:`With MkDocs </intro/getting-started-with-mkdocs>` |
-  :doc:`/choosing-a-site`
 
-* **Importing your existing documentation**:
-  :doc:`Import guide </intro/import-guide>`
+First time here?
+----------------
 
-Read the Docs feature overview
-------------------------------
+🚀 :doc:`/tutorial/index`
+  Read to take the first practical steps to getting started on Read the Docs?
 
-Learn more about configuring your automated documentation builds
-and some of the core features of Read the Docs.
+.. TODO: This next item needs its article to be finished in a separate PR
+.. https://github.com/readthedocs/readthedocs.org/pull/10071
 
-* **Overview of core features**:
-  :doc:`/custom-domains` |
-  :doc:`/versions` |
-  :doc:`/downloadable-documentation` |
-  :doc:`/server-side-search/index` |
-  :doc:`/analytics` |
-  :doc:`/pull-requests` |
-  :doc:`/build-notifications` |
-  :doc:`/user-defined-redirects` |
-  :doc:`/security-log`
+💡 :doc:`Why use a dedicated documentation platform? </integrations>`
+  Get an introduction to some of the most important reasons for using a *Documentation CI* and build *docs as code*.
 
-* **Connecting with GitHub, Bitbucket, or GitLab**:
-  :doc:`Connecting your VCS account </connected-accounts>`
+:doc:`🔠 All tutorials </tutorials/index>`
```

Comment:
😍

Reply:
I really liked seeing that, too, it felt like it opens an arena for more tutorial content